stripe/resource_stripe_price.go

```diff
@@ -556,14 +562,16 @@ func resourceStripePriceCreate(ctx context.Context, d *schema.ResourceData, m in
 		recurringMap := ToMap(recurring)
 		for k, v := range recurringMap {
 			switch {
-			case k == "interval" && ToString(v) != "":
+			case k == "interval":
 				params.Recurring.Interval = stripe.String(ToString(v))
 			case k == "interval_count" && ToString(recurringMap["interval"]) != "":
 				params.Recurring.IntervalCount = stripe.Int64(ToInt64(v))
-			case k == "aggregate_usage" && ToString(v) != "":
+			case k == "aggregate_usage":
 				params.Recurring.AggregateUsage = stripe.String(ToString(v))
-			case k == "usage_type" && ToString(v) != "":
+			case k == "usage_type":
 				params.Recurring.UsageType = stripe.String(ToString(v))
+			case k == "meter":
```

Comment:
Please change this to follow the same rules: `k == "meter" && ToString(v) != ""`.